### DISCUSSION TITLE
Missing pipeline subfolder

### DISCUSSION THREAD

**Discussion by lucacheli** [Azure-Player/azure.datafactory.tools#319] [Q&A] [answered]
I have a Pipeline Folder named "ETL_Detector_Photo", which includes a subfolder named "Iterative" with a few more pipelines in it.

Using the syntax for filtering:

+*.*@ETL_DataLake_Detector_Photo
or
+*.*@ETL_DataLake_Detector_Photo.*
or
+*.*@ETL_DataLake_Detector_Photo.Iterative

I am unable to publish this subfolder and the pipelines it contains on the destination ADF, while the parent folder and the pipelines it contains are correctly published.

Am I using the wrong syntax or there is some limitation on nested folders?

Thanks

**Comment by NowinskiK** [comment, marked as answer]
This is how location like "folder/subfolder" looks like in the code:
```
 "folder": {
            "name": "ExternalError/Subfolder"
        },
```
Therefore, you should try the following filtering syntax: `+pipeline.*@ExternalError/Subfolder`, 
or specifically in your case: `+pipeline.*@ETL_DataLake_Detector_Photo/Iterative`.

**Reply by lucacheli** [reply]
Hi Kamil,

it worked perfectly, I just did not know the correct syntax, nor had found it anywhere ...

Thank you
